### PR TITLE
chore: remove base chart

### PR DESCRIPTION
**Pull request by zirain**
**What type of PR is this?**

**What this PR does / why we need it**:

- remove CRD conversion
- merge into single chart
- will update doc later

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cmd/cluster-operator/README.md [hunk 1]
```diff
@@ -24,17 +23,16 @@ helm install -n cert-manager cert-manager jetstack/cert-manager --set installCRD
 
 ```
 # must run this after cert-manager is ready
-kubectl create namespace kurator-system
-helm install kurator-base out/charts/base-0.1.0.tgz -n kurator-system
-helm install kurator-cluster-operator out/charts/cluster-operator-0.1.0.tgz -n kurator-system
-helm install -f /root/capa-values.yaml -n kurator-system kurator-cluster-operator out/charts/cluster-operator-0.1.0.tgz
+helm install --create-namespace kurator-cluster-operator out/charts/cluster-operator-0.3-dev.tgz -n kurator-system
```
Comment (hzxuzhonghu) [inline cmd/cluster-operator/README.md:26]: can we put 0.3v-dev a into  var

File: cmd/cluster-operator/README.md [hunk 2]
```diff
@@ -24,17 +23,16 @@ helm install -n cert-manager cert-manager jetstack/cert-manager --set installCRD
...
 ```
 
 ## Cleanup
 
 ```
-helm uninstall kurator-cluster-operator -n kurator-system 
-helm uninstall kurator-base -n kurator-system
+helm uninstall kurator-cluster-operator -n kurator-system
 kubectl delete crd $(k get crds | grep cluster.x-k8s.io | awk '{print $1}')
+kubectl delete crd $(k get crds | grep kurator.dev | awk '{print $1}')
```
Comment (hzxuzhonghu) [inline cmd/cluster-operator/README.md:36]: wouldn't they deleted by helm uninstall?

File: manifests/charts/cluster-operator/crds/awsclustercontrolleridentities.infrastructure.cluster.x-k8s.io.yaml [hunk 3]
```diff
@@ -2,8 +2,8 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/kurator-serving-cert
     controller-gen.kubebuilder.io/version: v0.10.0
+  creationTimestamp: null
   labels:
     cluster.x-k8s.io/provider: infrastructure-aws
     cluster.x-k8s.io/v1alpha3: v1alpha3
```
Comment (hzxuzhonghu) [inline manifests/charts/cluster-operator/crds/awsclustercontrolleridentities.infrastructure.cluster.x-k8s.io.yaml:9]: nit: only leave the latest version

File: cmd/cluster-operator/README.md (hunk 2 above)
Comment (zirain) [reply cmd/cluster-operator/README.md:36]: this's expected https://helm.sh/docs/topics/charts/#limitations-on-crds

File: cmd/cluster-operator/README.md (hunk 1 above)
Comment (zirain) [reply cmd/cluster-operator/README.md:26]: this's a user face doc, so I think it's OK.

File: manifests/charts/cluster-operator/crds/awsclustercontrolleridentities.infrastructure.cluster.x-k8s.io.yaml (hunk 3 above)
Comment (zirain) [reply manifests/charts/cluster-operator/crds/awsclustercontrolleridentities.infrastructure.cluster.x-k8s.io.yaml:9]: done

File: cmd/crd-gen-tool/main.go [hunk 4]
```diff
@@ -170,7 +172,25 @@ func writeCRDs(outputDir string, resources kube.ResourceList) {
...
-		out, _ := yaml.Marshal(r.Object)
+		obj := r.Object.(*unstructured.Unstructured)
+		crd := &apiextv1.CustomResourceDefinition{}
+
+		_ = runtime.DefaultUnstructuredConverter.FromUnstructured(obj.Object, crd)
+
+		delete(crd.Annotations, "cert-manager.io/inject-ca-from")
+		crd.Spec.Conversion = nil
+
+		storagedVersions := make([]apiextv1.CustomResourceDefinitionVersion, 0, len(crd.Spec.Versions))
```
Comment (hzxuzhonghu) [inline cmd/crd-gen-tool/main.go:183]: nit: only one can be set storage

Comment (zirain) [reply cmd/crd-gen-tool/main.go]: done

**Review by hzxuzhonghu — APPROVED** [review]

**Pull request merged**